websites/E/EternalNetwork TM/presence.ts

```diff
@@ -79,16 +79,16 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/memberlist.php"))
-				presenceData.state = "Sneaking into member list !";
+				presenceData.state = "Sneaking into member list";
```

Comment:
This sounds wrong on some many levels

Reply:
Fixed, I hope "Viewing an admin panel" does make sense and also follows the line of the verb - viewing.